lib/phoenix_live_view/channel.ex

```diff
@@ -508,6 +509,17 @@ defmodule Phoenix.LiveView.Channel do
     {:stop, :shutdown, :no_session}
   end
 
+  defp verify_flash(endpoint, verified, params) do
+    flash_token = params["flash"]
+    verified_flash = verified[:flash]
+
+    cond do
+      flash_token -> Phoenix.LiveView.Flash.verify!(endpoint, flash_token)
```

Comment:
We probably don't want to raise on verify. For example, imagine that you do not a new deploy with a new secret key base.